### ISSUE TITLE
Study is not Load in Study Browser. While Loading the Modality "SM"

### ISSUE THREAD

**Issue by Malik-Adil** [OHIF/Viewers#626]
When I Open the Study having the modality SM. Study Browser shows the black screen only as shown in the screen Short

![Screenshot from 2019-07-05 09-56-36](https://user-images.githubusercontent.com/20414066/60699000-689bac00-9f0b-11e9-97f9-3c2f818ac232.png)

When Select the Tool from the tool list. The tool becomes active but I am not able to use in the Viewer?

![Screenshot from 2019-07-05 09-57-17](https://user-images.githubusercontent.com/20414066/60699050-9c76d180-9f0b-11e9-8284-49c3826ffd23.png)

When I Drag the mouse in the Viewer Only Zoom Work.

**Comment by swederik** [comment]
Yes the tools do not currently work. The whole-slide microscopy extension uses https://github.com/dcmjs-org/dicom-microscopy-viewer under the hood, and we have not set up the commands to allow tool usage on this viewport yet.

We would welcome any pull requests to improve the microscopy extension because we do not currently have funding to improve this functionality. If your company is willing to sponsor development, that would also go a long way.

Essentially you would need to add the equivalent modules to the microscopy extension to enable tools via the `activateDrawInteraction` API method (https://github.com/dcmjs-org/dicom-microscopy-viewer/blob/c706f918c0387c40f34cff922d6200e18d43b9ce/src/api.js#L765):

https://github.com/OHIF/Viewers/blob/master/extensions/ohif-cornerstone-extension/src/commandsModule.js

https://github.com/OHIF/Viewers/blob/master/extensions/ohif-cornerstone-extension/src/toolbarModule.js

**Comment by swederik** [comment]
Just noticed that I did not address the initial issue:
- Thumbnails are not loaded for microscopy-extension images. That is another issue that is up for grabs.
